packages/react/src/Link/Link.test.tsx

```diff
@@ -0,0 +1,13 @@
+import { render } from '@testing-library/react'
+import { Link } from './Link'
+import '@testing-library/jest-dom'
+
+describe('Link', () => {
+  it('renders a link role element', () => {
```

Comment:
I would avoid nested `it` statement. They should always have an assertion. This feels like composing a test case